Makefile

```
@@ -90,12 +92,13 @@ test-unit:
 	
 .PHONY: test-integration
 test-integration:
-	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); fi; \
-	if [[ -f $(PWD)/$(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(PWD)/$(TARGET_KUBECONFIG); fi; \
-	if [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1=$(PWD)/$(MACHINECLASS_V1); fi; \
-	if [[ -f $(PWD)/$(MACHINECLASS_V2) ]]; then export MACHINECLASS_V2=$(PWD)/$(MACHINECLASS_V2); fi; \
+	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); elif [[ -f $(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(CONTROL_KUBECONFIG);else echo "No such file exists for CONTROL_KUBECONFIG";exit 1; fi; \
+	if [[ -f $(PWD)/$(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(PWD)/$(TARGET_KUBECONFIG); elif [[ -f $(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(TARGET_KUBECONFIG);else echo "No such file exists for TARGET_KUBECONFIG";exit 1; fi; \
+	if [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1=$(PWD)/$(MACHINECLASS_V1);elif [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1="$(MACHINECLASS_V1)"; fi; \
```

From context, I think it should be as follows
```suggestion
	if [[ -f $(PWD)/$(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1=$(PWD)/$(MACHINECLASS_V1);elif [[ -f $(MACHINECLASS_V1) ]]; then export MACHINECLASS_V1="$(MACHINECLASS_V1)"; fi; \
```